# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openaptx/sys-ffmpeg.patch
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- openaptx-4160c018fe120ddaba378d54eb140386a981dbb7/src/Makefile.am.vanilla	2018-06-04 16:20:21.308747372 +0200
+++ openaptx-4160c018fe120ddaba378d54eb140386a981dbb7/src/Makefile.am	2018-06-04 16:20:53.212747749 +0200
@@ -11,9 +11,7 @@
 	bt-aptx-stub-sonar.c \
 	bt-aptx-ffmpeg.c
 
-libbt_aptx_la_LDFLAGS = -no-undefined --whole-archive\
-$(top_builddir)/libavcodec/libavcodec.a\
-$(top_builddir)/libavutil/libavutil.a
+libbt_aptx_la_LDFLAGS = -no-undefined -lavcodec -lavutil
 	
 bt-aptx-stub-sonar.c: $(srcdir)/sonar.aptx
 	echo " \
